src/server.ts

```diff
@@ -41,10 +41,10 @@ export function makeServer(appState: AppState): Express {
       return next(err)
     }
 
-    const response: ApiErrorResponse = {
+    const response: ServerErrorResponse = {
       success: false,
       message: err.message,
-      error: err.stack
+      stack: err.stack
```

Comment:
Do we really want to reveal this information? I guess it's fine, just wondering what the reason was.

Reply:
It's been used for development, but there is a task to move this out of the response for production and only log the stack trace.